pkg/model/manifest.go

```diff
@@ -222,6 +224,14 @@ func (m Manifest) LocalPaths() []string {
 	return sliceutils.DedupedAndSorted(paths)
 }
 
+func (m Manifest) WithLabels(labels map[string]string) Manifest {
```

Comment:
i think what you probably want to do is call this function from tiltfile_state.go, instead of assigning Labels directly

Reply:
yes! this should be fixed now. thanks for your help. 🎉